Java/libraries/recognizers-text-datatypes-timex-expression/src/main/java/com/microsoft/recognizers/datatypes/timex/expression/TimexDateHelpers.java

```diff
@@ -119,4 +119,8 @@ public static LocalDateTime dateOfNextDay(DayOfWeek day, LocalDateTime reference
 
         return result;
     }
+
+    public static Integer getUSDateOfWeekValue(DayOfWeek dayOfWeek) {
```

Comment:
I'm not seeing these changes in the PR 2457, or in the C# file in that branch, but I assume this was necessary

Reply:
This is a disparity between how DayOfWeek (which I noticed that this should be renamed to `getUSDayOfweek`) works in C# vs Java.

C# has Sunday as 0
Java has Sunday as 7